damnit/gui/main_window.py

```diff
@@ -598,42 +582,39 @@ def inspect_data(self, index):
             log.warning(f"Unrecognized variable: '{quantity}'")
             return
 
-        if is_image:
-            try:
-                image = variable.ndarray()
-            except KeyError:
-                log.warning("'{}' not found in {}...".format(quantity, variable.file))
-                return
+        try:
+            data = variable.xarray()
+        except KeyError:
+            log.warning(f'"{quantity}" not found in {variable.file}...')
+            return
 
+        if is_image or data.ndim == 2:
```

Comment:
`is_image` here is telling us whether what we display in a cell is a PNG thumbnail.  I think this is now irrelevant - we only need to check the array that we've loaded from the HDF5 file. But we should also check for 3D (RGB) arrays, in that case.

Reply:
I think those are the only cases we need. :+1:

Reply:
good there are tests :stuck_out_tongue:  , figures are converted to `rgba` so:
`if (data.ndim == 2) or (data.ndim == 3 and data.shape[-1] in (3, 4))`

Reply:
Ooh yeah, I'm glad the tests caught that.